offlinevalidator/offlinevalidator.go

```diff
@@ -96,6 +104,38 @@ func Validate(context *context.Context) bool {
 	return valid
 }
 
+func findingAllowedValues(rawTemplate []byte, logger *logger.Logger) (valid bool) {
```

Comment:
We should rather base it on `goformation` parsed tree, than parsing lines manually.